### PR TITLE
AB#108138 enable sw fallback

### PR DESCRIPTION
**Pull request by jngd**
Enable SW fallback

**Issue**

Calling to **reconfigure_codec**  we are stopping/starting a fresh created mediacodec in order to reassign the surface. This process is not working on RSplus and also is not needed for our use case since we are using one player for each stream so we change the method used to use configure_codec that is not stoping the decoder. 

**Tests**

Working fine on all targets that includes:

- SMART Moose scaler
- Dell TPV D7523QT
- QSeries
- NT
- RSplus

### CODE REVIEW

File: ijkmedia/ijkplayer/android/pipeline/ffpipenode_android_mediacodec_vdec.c [hunk 1]
```diff
@@ -2060,7 +2060,7 @@ IJKFF_Pipenode *ffpipenode_create_video_decoder_from_android_mediacodec(FFPlayer
     }
 
     jsurface = ffpipeline_get_surface_as_global_ref(env, pipeline);
-    ret = reconfigure_codec_l(env, node, jsurface);
+    ret = configure_codec_l(env, node, jsurface);
```
Comment (franNadalesDN) [inline ijkmedia/ijkplayer/android/pipeline/ffpipenode_android_mediacodec_vdec.c:2065]: I have no context to see if this makes the trick or not.

Comment (jngd) [reply ijkmedia/ijkplayer/android/pipeline/ffpipenode_android_mediacodec_vdec.c:2065]: description updated

Comment (franNadalesDN) [reply ijkmedia/ijkplayer/android/pipeline/ffpipenode_android_mediacodec_vdec.c:2065]: I would add here a comment to mention original implementation uses reconfigure. If later we see we can improve something here is good to have such information.

**Review by franNadalesDN — COMMENTED** [review]
Just add a comment explaining why that part has been changed from original implementation.

File: ijkmedia/ijkplayer/android/pipeline/ffpipenode_android_mediacodec_vdec.c (hunk 1 above)
Comment (jngd) [reply ijkmedia/ijkplayer/android/pipeline/ffpipenode_android_mediacodec_vdec.c:2065]: Done

**Review by franNadalesDN — APPROVED** [review]
Well done!

**Pull request merged**